### PR TITLE
Local dev settings

### PR DESCRIPTION
**Pull request by EDsCODE**
updated config.py with nick so that config.py uses local environment variables (DATABASE_USER, DATABASE_PASSWORD) when starting postgres

**my styler in VSCODE shifted the spacing of a lot of lines in the file

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -161,29 +143,21 @@
 REDIS_URL = 'redis://' + specific_parser['REDIS']['URI']
 
 DATABASE_USER = os.environ.get("DATABASE_USER") or specific_parser['DATABASE'].get('user') \
```
Comment (EDsCODE) [inline config.py]: main change added to DATABASE_USER to parse environment variable

File: config.py [hunk 2]
```diff
@@ -161,29 +143,21 @@
...
 
 DATABASE_HOST = specific_parser['DATABASE']['host']
 
 DATABASE_NAME = specific_parser['DATABASE'].get('database') \
-    or common_parser['DATABASE']['database']
+                or common_parser['DATABASE']['database']
 
-DATABASE_PASSWORD = os.environ.get(
-    "DATABASE_PASSWORD") or specific_parser['DATABASE']['password']
+DATABASE_PASSWORD = os.environ.get("DATABASE_PASSWORD") or specific_parser['DATABASE']['password']
```
Comment (EDsCODE) [inline config.py]: main change added to DATABASE_PASSWORD to parse environment variable

**Comment by EDsCODE** [comment]
I restored the files format to match the original spacing in the file and noted where the main changes were made for the environment variable parsing

**Comment by enjeyw** [comment]
> **my styler in VSCODE shifted the spacing of a lot of lines in the file

Yeah this is a known issue we have. Solution tbc lol

**Pull request merged**